### PR TITLE
[DE] Add some set timer variations

### PR DESCRIPTION
**Pull request by jpbede**
Add some variations for setting a timer e.g. "Stelle Timer auf eine Stunde" or "Stelle einen Pizza Timer auf eine Stunde"

### CODE REVIEW

File: sentences/de/homeassistant_HassStartTimer.yaml [hunk 1]
```diff
@@ -7,11 +7,13 @@ intents:
           - "Timer für <timer_duration>"
```
Comment (mib1185) [inline sentences/de/homeassistant_HassStartTimer.yaml:7]:
```suggestion
          - "Timer (auf|für) <timer_duration>"
```

File: sentences/de/homeassistant_HassStartTimer.yaml [hunk 2]
```diff
@@ -7,11 +7,13 @@ intents:
...
           - "<timer_duration> Timer für {timer_name:name}"
           - "Timer für <timer_duration> namens {timer_name:name}"
```
Comment (mib1185) [inline sentences/de/homeassistant_HassStartTimer.yaml:9]:
```suggestion
          - "Timer (auf|für) <timer_duration> namens {timer_name:name}"
```

File: sentences/de/homeassistant_HassStartTimer.yaml [hunk 3]
```diff
@@ -7,11 +7,13 @@ intents:
...
+          - "{timer_name:name} Timer (auf|für) <timer_duration>"
           - "<timer_set>[ einen] <timer_duration> Timer"
-          - "<timer_set>[ einen] Timer für <timer_duration>"
+          - "<timer_set>[ einen] Timer (auf|für) <timer_duration>"
           - "<timer_set>[ einen] <timer_duration> Timer namens {timer_name:name}"
           - "<timer_set>[ einen] Timer namens {timer_name:name} für <timer_duration>"
           - "<timer_set>[ einen] Timer für <timer_duration> namens {timer_name:name}"
+          - "<timer_set>[ einen] {timer_name:name} Timer (auf|für) <timer_duration>"
```
Comment (mib1185) [inline sentences/de/homeassistant_HassStartTimer.yaml:16]:
```suggestion
          - "<timer_set>[ einen][ Timer (auf|für)] <timer_duration> Timer namens {timer_name:name}"
          - "<timer_set>[ einen][ Timer namens] {timer_name:name} (auf|für) <timer_duration>"
```

File: tests/de/homeassistant_HassStartTimer.yaml [hunk 4]
```diff
@@ -4,6 +4,7 @@ tests:
   - sentences:
       - "Starte einen 2 Stunden Timer"
       - "Stelle Timer für 2 Stunden"
+      - "Stelle einen Timer auf 2 Stunden"
       - "Erstelle 2 Stunden Timer"
       - "2 Stunden Timer"
       - "Timer für 2 Stunden"
```
Comment (mib1185) [inline tests/de/homeassistant_HassStartTimer.yaml:10]:
```suggestion
      - "Starte einen 2 Stunden Timer"
      - "Stelle Timer auf 2 Stunden"
      - "Stelle Timer für 2 Stunden"
      - "Stelle einen Timer auf 2 Stunden"
      - "Stelle einen Timer für 2 Stunden"
      - "Erstelle 2 Stunden Timer"
      - "2 Stunden Timer"
      - "Timer auf 2 Stunden"
      - "Timer für 2 Stunden"
```

File: tests/de/homeassistant_HassStartTimer.yaml [hunk 5]
```diff
@@ -16,6 +17,7 @@ tests:
   - sentences:
       - "Starte einen 1 Stunde Timer"
       - "Stelle Timer für 1 Stunde"
+      - "Stelle einen Timer auf 1 Stunde"
       - "Erstelle 1 Stunde Timer"
       - "1 Stunde Timer"
       - "Timer für 1 Stunde"
```
Comment (mib1185) [inline tests/de/homeassistant_HassStartTimer.yaml:23]:
```suggestion
      - "Starte einen 1 Stunde Timer"
      - "Stelle Timer auf 1 Stunde"
      - "Stelle Timer für 1 Stunde"
      - "Stelle einen Timer auf 1 Stunde"
      - "Stelle einen Timer für 1 Stunde"
      - "Erstelle 1 Stunde Timer"
      - "1 Stunde Timer"
      - "Timer auf 1 Stunde"
      - "Timer für 1 Stunde"
```

File: tests/de/homeassistant_HassStartTimer.yaml [hunk 6]
```diff
@@ -84,6 +86,10 @@ tests:
       - "5 Minuten Timer für Pizza"
       - "Stelle Timer namens Pizza für 5 Minuten"
       - "Timer für 5 Minuten namens Pizza"
+      - "Pizza Timer für 5 Minuten"
+      - "Pizza Timer auf 5 Minuten"
+      - "Stelle einen Pizza Timer für 5 Minuten"
+      - "Stelle Pizza Timer auf 5 Minuten"
```
Comment (mib1185) [inline tests/de/homeassistant_HassStartTimer.yaml:92]:
```suggestion
      - "5 Minuten Timer für Pizza"
      - "Stelle Timer namens Pizza auf 5 Minuten"
      - "Stelle Timer namens Pizza für 5 Minuten"
      - "Timer auf 5 Minuten namens Pizza"
      - "Timer für 5 Minuten namens Pizza"
      - "Pizza Timer auf 5 Minuten"
      - "Pizza Timer für 5 Minuten"
      - "Stelle einen Pizza Timer auf 5 Minuten"
      - "Stelle einen Pizza Timer für 5 Minuten"
      - "Stelle Pizza Timer auf 5 Minuten"
      - "Stelle Pizza Timer für 5 Minuten"
```

**Review by mib1185 — DISMISSED** [review]
Hi @jpbede
thanks for this nice addition :heart: 
Let us make the `auf|für` variation consistent for all sentences. Updating the other test sentences would also be great, thx :+1:

**Review by frenck — APPROVED** [review]
Thanks, @jpbede 👍 

../Frenck

**Pull request merged**